tests/sentry/processing/realtime_metrics/test_redis.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+import time
+from typing import TYPE_CHECKING, Any, Dict
+
+import pytest
+
+from sentry.processing import realtime_metrics  # type: ignore
+from sentry.processing.realtime_metrics.redis import RedisRealtimeMetricsStore  # type: ignore
+from sentry.utils import redis
+
+if TYPE_CHECKING:
+    from typing import Callable
+
+    def _fixture(func: Callable[..., Any]) -> Callable[..., None]:
+        ...
+
+    pytest.fixture = _fixture
+
+
+@pytest.fixture
+def config() -> Dict[str, Any]:
+    return {
+        "cluster": "default",
+        "counter_bucket_size": 10,
+        "counter_ttl": datetime.timedelta(milliseconds=400),
+    }
+
+
+@pytest.fixture
+def redis_cluster(config: Dict[str, Any]) -> redis._RedisCluster:
+    cluster, options = redis.get_cluster_from_options(
+        "TEST_CLUSTER", config, cluster_manager=redis.redis_clusters
+    )
+    return cluster
+
+
+@pytest.fixture
+def store(config: Dict[str, Any]) -> RedisRealtimeMetricsStore:
+    return RedisRealtimeMetricsStore(**config)
+
+
+def test_default() -> None:
+    realtime_metrics.increment_project_event_counter(17, 1234)
+
+
+def test_increment_project_event_counter(
+    store: RedisRealtimeMetricsStore, redis_cluster: redis._RedisCluster
+) -> None:
+    store.increment_project_event_counter(17, 1147)
+    assert redis_cluster.get("symbolicate_event_low_priority:counter:10:17:1140") == "1"
+    time.sleep(0.5)
+    assert redis_cluster.get("symbolicate_event_low_priority:counter:10:17:1140") is None
+
+
+def test_increment_project_event_counter_twice(
```

Comment:
i assume this is more concerned with checking the TTL refresh behaviour if there are multiple inserts into the same bucket; perhaps it would be better to rename this to reflect that? based off of the test name it sounds like this is testing a very specific case of `test_increment_project_event_counter_multiple`.

i suppose the difference between the two could also be testing multi insertion into the same bucket vs different buckets?